.github/workflows/build.yml

```diff
@@ -179,6 +179,17 @@ jobs:
       run: |
         BUILD_MODE=${{ matrix.build_type }} ./test.sh test-build-and-run-sample-c-appln-with-LOC-ELF
 
+    #! -------------------------------------------------------------------------
+    #  This test will build-and-run -ALL- client-server perf-test, with 1-client
+    #  each sending 1 Million messages to the server. This test-method is
+    #  turned OFF for normal CI-jobs. Enable this, and delete the rest of the
+    #  commands, to validate perf-related changes in a CI-build environment.
+    #  Just run with 1 client as CI-VMs seem to not be very high-powered CPUs.
+    #! -------------------------------------------------------------------------
+    # - name: test-run-all-client-server-perf-tests
```

Comment:
Left this test-method in place, so we can comment out the rest and just enable this one, to get a quick turnaround in CI-jobs when testing other perf changes.